src/main/java/org/jitterbit/connector/dropbox/activities/BaseDropboxActivity.java

```diff
@@ -32,7 +32,7 @@ public BaseDropboxActivity() {
   }
 
   @Override
-  public ActivityRequestResponseMetaData 
+  public ActivityRequestResponseMetaData
```

Comment:
Did you rebuild the connector after this change? After changing source code, always need to rebuild, as we don't have any automatic builds that are testing this.

Reply:
I did, but it shows no changes to files in the `target` folder. I think this makes sense, since my changes didn't have to do with the building portion of the code.

If you think I should see changes here, let's go over why this might be.